src/modules/manual_control/ManualControl.cpp

```diff
@@ -67,6 +67,16 @@ void ManualControl::Run()
 	perf_begin(_loop_perf);
 	perf_count(_loop_interval_perf);
 
+	processInput(hrt_absolute_time());
+
+	// reschedule to detect timeouts
+	ScheduleDelayed(200_ms);
```

Comment:
Hardly matters in any real usage, but this could be COM_RC_LOSS_T.

Reply:
I thought about it but then the scheduling depends on a parameter that can be changed potentially to an unreasonable value even in air. Is that a good idea? Does it need additional sanity checking then to not screw up the entire system?